src/scripts/main.js

```diff
@@ -2,7 +2,28 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+for (const person of people) {
+  const table = document.querySelector('tbody');
 
-// write your code here
+  const row = document.createElement('tr');
+
+  table.append(row);
+
+  row.insertAdjacentHTML('afterbegin', `
+    <th>${person.name}</th>
```

Comment:
```suggestion
    <td>${person.name}</td>
```
`th` tag used for table head